src/use_event_listener/index.jsx

```diff
@@ -1,18 +1,26 @@
-import { useEffect } from 'react';
+import { useEffect, useRef } from 'react';
 
 export const useEventListener = (
   ref,
   type,
-  listener,
-  options = { passive: true },
+  handler,
+  { passive, capture, once } = { passive: true },
 ) => {
+  const node = ref.current;
+  const handlerRef = useRef(null);
+
+  handlerRef.current = handler;
```

Comment:
I am not sure about this line

Found a similar implementation but they use `useEffect` to set the handler ref. https://github.com/donavon/use-event-listener/blob/develop/src/index.js#L13

Reply:
it is re-executed every render anyway, same difference